hermes/bin/src/runtime_context.rs

```diff
@@ -22,6 +25,9 @@ pub(crate) struct HermesRuntimeContext {
     /// App Virtual file system
     #[allow(dead_code)]
     vfs: Arc<Vfs>,
+
+    /// Runtime WASM context.
+    wasi_ctx: WasiContext,
```

Comment:
I think this fs runtime extension resources manager should be a static variable as we already done in other runtime extensions like sqlite, crypto.